<ui version="4.0" >
 <class>LoginBox</class>
 <widget class="QWidget" name="LoginBox" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>LoginBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelHost" >
     <property name="text" >
      <string>&amp;Host:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>host</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="host" />
   </item>
   <item row="0" column="3" >
    <widget class="QLabel" name="labelPort" >
     <property name="text" >
      <string>P&amp;ort:</string>
     </property>
     <property name="buddy" >
      <cstring>port</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="4" >
    <widget class="QSpinBox" name="port" >
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>65535</number>
     </property>
     <property name="value" >
      <number>80</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="labelUser" >
     <property name="text" >
      <string>&amp;User:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>user</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="user" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="labelPassword" >
     <property name="text" >
      <string>&amp;Password:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>password</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLineEdit" name="password" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="labelRepeat" >
     <property name="text" >
      <string>&amp;Confirm:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>repeat</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QLineEdit" name="repeat" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>59</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" >
    <widget class="QCheckBox" name="remember" >
     <property name="toolTip" >
      <string>Remember password across sessions</string>
     </property>
     <property name="whatsThis" >
      <string>Checking this field will cause your password to be saved for future use.</string>
     </property>
     <property name="text" >
      <string>&amp;Remember password</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5" >
    <widget class="QGroupBox" name="proxy" >
     <property name="toolTip" >
      <string>Enable network proxy</string>
     </property>
     <property name="whatsThis" >
      <string>Enabling this group will allow you to enter data for a network proxy if your Internet provider requires one.</string>
     </property>
     <property name="title" >
      <string>Pro&amp;xy</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <property name="margin" >
       <number>2</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Host:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="proxyHost" />
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QSpinBox" name="proxyPort" >
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buttonSymbols" >
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>65535</number>
        </property>
        <property name="value" >
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>User:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="proxyUser" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Password:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="proxyPassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="labelProxyRepeat" >
        <property name="text" >
         <string>Confirm:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="proxyRepeat" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
